src/assets/scripts/client/App.js

```diff
@@ -62,6 +63,9 @@ export default class App {
      */
     createHandlers() {
         this.eventBus.on(EVENT.PAUSE_UPDATE_LOOP, this.updateRun);
+        this.onLoadStorageIcaoFailureHandler = this.onLoadStorageIcaoFailure.bind(this);
+        this.onLoadAirlinesAndAircraftHandler = this.onLoadAirlinesAndAircraft.bind(this);
+        this.setupChildrenHandler = this.setupChildren.bind(this);
```

Comment:
wouldn't we want our handlers to be private?

Reply:
this is the top level file, so they are all in a sense public.  the only exception is `_init()` which has both a private and public version in this file.